runtime-sdk/modules/contracts/src/abi/oasis/crypto.rs

```diff
@@ -91,33 +90,68 @@ impl<Cfg: Config> OasisV1<Cfg> {
                 };
                 gas::use_gas(ctx.instance, cost)?;
 
-                ctx.instance
-                    .runtime()
-                    .try_with_memory(|memory| -> Result<_, wasm3::Trap> {
-                        let key = get_key(kind, key, &memory).map_err(|err| {
-                            ec.aborted = Some(Error::CryptoMalformedPublicKey);
-                            err
-                        })?;
-                        let message = Region::from_arg(message)
+                let rt = ctx.instance.runtime();
```

Comment:
nothing here changed. I just split `rt` into a var to reduce one layer of indentation for readability